### PR TITLE
Token documentation

### PR DESCRIPTION
**Pull request by rurikoaraki**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Adding documentation for how to change tokens using FURN

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [x] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: docs/pages/Theming/Tokens/Overrides.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    }
+  },
+);
+```
+
+```ts
+
+```
+
+## Overriding Component Tokens
```
Comment (Saadnajmi) [inline docs/pages/Theming/Tokens/Overrides.md:89]: So the name has changed from "control tokens"? Good to know! :)

**Review by Saadnajmi — APPROVED** [review]
@mischreiber fyi, in case this is useful for comparing how overrides work between FluentUI Apple and FluentUI React Native.

File: docs/pages/Theming/Tokens/Overrides.md (hunk 1 above)
Comment (rurikoaraki) [reply docs/pages/Theming/Tokens/Overrides.md:89]: I think that's what they called it? I don't know if we have official support for it though.

File: docs/pages/Theming/Tokens/Overrides.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+# Overriding Tokens
+
+This document explains how to override tokens from the Fluent Design System using FURN's Theming system.
+
+## Overriding Global Tokens
+
+When components using tokens were developed, it was assumed that global tokens would not be allowed to change. Because of this, any global tokens that are used directly by components, such as border radius tokens or shared colors, will not have changed reflected within components even if they are overridden in a codebase.
```
Comment (joannaquu) [inline docs/pages/Theming/Tokens/Overrides.md:7]: typo: changes instead of changed?

**Pull request merged**